jobs/2017-04-28-frontend-branding-for-composerci.md

```diff
@@ -5,15 +5,15 @@ date_posted: '2017-04-28'
 layout: jobs
 organization: ComposerCI
 org_url: 'mailto:composerci@miguelpiedrafita.com'
-title: Frontend & branding for ComposerCI
+title: Branding for ComposerCI
 role: Designer
 compensation: gratis
-description: "ComposerCI is a Composer Continous Integration tool, that send a Pull Request every time a Composer dependency you use is updated.\r\nWe are starting the development, so we can use any frontend framework you are familiar with!"
-skills: "If you design the frontend, scss, javascript and html files should be delivered.\r\nIf you do the branding, svg & png files for the logo, the font and main colors."
+description: "ComposerCI is a Composer Continous Integration tool, that send a Pull Request every time a Composer dependency you use is updated."
+skills: "Svg & png files for the logo, the font and main colors should be delivered."
 how_to_apply:
   - Send a mail to composerci@miguelpiedrafita.com
   - ''
-tags: 'frontend, branding, logo, design, gratis, free'
+tags: 'branding, logo, design, gratis, free'
 date: '2017-04-28T13:28:04.234Z'
 ---
 undefined
```

Comment:
Would be nice if we could remove this "undefined".
Is it intended? Since it passed 2 revisions